src/components/CandidateNameSubtitle/CandidateNameSubtitle.tsx

```diff
@@ -21,7 +22,7 @@ export function CandidateNameSubtitle({ id, name }: Props) {
                 lineHeight: '28px',
             }}
         >
-            Candidate: {typeof id !== 'number' ? name : <Link href={pageHrefs.candidate(id)}>{name}</Link>}
+            {tr('Candidate')}: {typeof id !== 'number' ? name : <Link href={pageHrefs.candidate(id)}>{name}</Link>}
```

Comment:
just use nullable

Reply:
Why? There will be name of candidate or link to his interview